scenarioo-client/test/spec/build/build.controller.spec.ts

```diff
@@ -17,48 +17,56 @@
 
 'use strict';
 
-describe('BuildController', function () {
+import {Observable} from "rxjs";
+declare var angular: angular.IAngularStatic;
 
-    var $location, $httpBackend, TestData, $scope, BuildController;
+describe('BuildController', () => {
+
+
+    let $location, TestData, $scope, BuildController;
```

Comment:
tslint should fail on that actually. Use seperate declearion per variable and const where possible

Reply:
This is the way how it is implemented in many other specs as well. So I just sticked to that pattern. e.g. `ScenarioController`